cukedoctor-converter/src/main/java/com/github/cukedoctor/builder/AttributesBuilderImpl.java

```diff
@@ -134,6 +134,14 @@ public AttributesBuilder allowUriRead(Boolean value) {
         return this;
     }
 
+    @Override
+    public AttributesBuilder dataUri(Boolean value) {
+        if (value) {
```

Comment:
Shouldn't we make it possible to turn it off? if then we need set whenever it's different than null

```suggestion
        if (value != null) {
```

Reply:
Or we can change `allowUriRead` to use the same approach here so we avoid writing the attribute:
https://github.com/rmpestano/cukedoctor/blob/d13a955818406380e79185732c9f09a60c9c44ad/cukedoctor-converter/src/main/java/com/github/cukedoctor/builder/AttributesBuilderImpl.java#L131-L133

Reply:
I only want to set it if the value is "true", really. I know you can do :!data-uri: but that seems a bit wasteful since AsciiDoc has this attribute off by default.

Reply:
Yea, can we do the same for `allowUriRead`? 

Thanks, again!

Reply:
I'd rather do it in a separate PR, if you'd care to create an issue. In fact, I could review all the boolean attributes we support and ensure we only set if explicitly specified AND what is specified is not the default.

Reply:
Sure, that also works